sccp.c

```diff
@@ -88,6 +88,21 @@ static struct ast_channel *channel_tech_requester(const char *type, struct ast_f
 		break;
 	}
 
+	if (!line) {
+		return NULL;
```

Comment:
I think channel should be returned here like it used to before logic was added to this function